sdk/ml/azure-ai-ml/azure/ai/ml/entities/_featurestore_entity/data_column.py

```diff
@@ -0,0 +1,54 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=redefined-builtin,disable=unused-argument
+
+from typing import Dict
+
+from azure.ai.ml._restclient.v2023_02_01_preview.models import IndexColumn, FeatureDataType
+
+from azure.ai.ml.entities._mixins import RestTranslatableMixin
+from azure.ai.ml._utils._experimental import experimental
+
+from .data_column_type import DataColumnType
+
+DataColumnTypeMap: Dict[DataColumnType, FeatureDataType] = {
```

Comment:
We also should have boolean type support here, as we want to be aligned to mlflow types
https://mlflow.org/docs/latest/python_api/mlflow.types.html#mlflow.types.DataType

Reply:
updated MFE, its bool in the spec. 
I can change it to boolean.
cc: @qjxu

Reply:
Please make it consistent with MFE contract. Thanks.

Reply:
okay, will update to boolean in next PR